Jesse-Kramer/tumi-mundo
# Story component

## What does this change?

- Added a **Story Component** that dynamically renders a list of stories.  
- Each story includes:
  - Title, summary, language flag, playtime, and interactive icons (e.g., download and add to playlist).  
  - Responsive layout with support for line-clamping to prevent text overflow.  
- Designed a flexible and clean UI using CSS grid and flexbox.  

## How Has This Been Tested?


![Story Component Example](https://github.com/user-attachments/assets/56998a6e-b8fa-4606-abcb-8bfd6de534a6)

## Images

![Rendered Story Component](https://github.com/user-attachments/assets/8a6da358-9e80-451e-8caf-749ea2f1738c)

## How to review

1. Pull the branch locally.  
2. Run the app and navigate to the page containing the story component.  
3. Verify:
   - Stories render correctly with all provided data.  
   - Line-clamping works as expected for longer text.  
   - UI layout maintains integrity across screen sizes (test responsiveness).  
   - Icons are functional and visually aligned.
